package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "lovelace-horizon-card",
   "version": "1.1.0",
+  "packageManager": "yarn@4.5.1",
   "description": "Successor to Sun Card - Visualize the position of the sun over the horizon",
   "type": "commonjs",
   "main": "dist/index.js",
   "scripts": {
     "dev": "rollup -c rollup.config.dev.mjs --watch",
-    "build": "rimraf dist && yarn lint && yarn rollup",
+    "build": "rm -fr dist && yarn lint && yarn rollup",
```

Comment:
Modern yarn has its own shell, to guarantee that things like `rm -fr` works the same across all platforms, including Windows.